app/components/AppBreadCrumb.tsx

```diff
@@ -46,8 +46,11 @@ export default function AppBreadCrumb() {
   const crumbs = useCrumbs(pathName);
   const isDetailsPage = crumbs?.length > 1;
 
+  if (!isDetailsPage) {
+    return null;
+  }
```

Comment:
fixes a bug where we had an empty `row` in the DOM that was inserting some vertical margin